package.json

```diff
@@ -2008,6 +2008,17 @@
                     "tags": [
                         "experimental"
                     ]
+                },
+                "jupyter.experimental.kernelPickerType": {
+                    "type": "string",
+                    "enum": [
+                        "Stable",
+                        "OnlyOneTypeOfKernel",
+                        "Insiders"
```

Comment:
I think you can make use of `enumDescriptions` to add descriptions for each item, its a new VSC API.

Reply:
Ohh, nice. Will change to use that.